gufe/settings/models.py

```diff
@@ -0,0 +1,143 @@
+    for more details.
+    """
+    scale12: float = 0.0
+    scale13: float = 0.0
+    scale14: float = 0.833333
+    scale15: float = 1.0
+
+
+class VdWSettings(SettingsBaseModel):
+    """Settings for van der Waals force.
+
+    See the `SMIRNOFF vdW specification <https://openforcefield.github.io/standards/standards/smirnoff/#vdw>`_
+    for more details.
+    """
+
+    combining_rules: Literal["Lorentz-Berthelot"]
+    potential: Literal["Lennard-Jones-12-6"]
+    scale: vdWScale
+    long_range_dispersion: Literal["isotropic"]
+    cutoff: FloatQuantity["angstrom"] = Field(9.0, description="Default units are angstroms")
+    switch_width: FloatQuantity["angstrom"] = Field(1.0, description="Default units are angstroms")
+    method: Literal["cutoff"]
+
+
+class ElectrostaticSettings(SettingsBaseModel):
+    """Settings for electrostatics.
+
+    See the `SMIRNOFF Electrostatics specification <https://openforcefield.github.io/standards/standards/smirnoff/#electrostatics>`_
+    for more details.
+
+    """
+
+    # Tricky since this allows functions https://openforcefield.github.io/standards/standards/smirnoff/#electrostatics
+    periodic_potential: Union[Literal["Ewald3D-ConductingBoundary"], str]
+    nonperiodic_potential: Union[Literal["Coulomb"], str]
+    exception_potential: Union[Literal["Coulomb"], str]
+    scale: ElectrostaticScale
+    cutoff: FloatQuantity["angstrom"] = Field(None, description="Default units are in angstroms")
+    switch_width: FloatQuantity["angstrom"] = Field(None, description="Default units are in angstroms")
+    solvent_dielectric: Optional[float]
+
+
+class GBSASettings(SettingsBaseModel):
+    """Settings for Generalized-Born surface area (GBSA) implicit solvent parameters.
+
+    See the `SMIRNOFF GBSA specification <https://openforcefield.github.io/standards/standards/smirnoff/#gbsa>`_
+    for more details.
+    """
+
+    gb_model: str = "OBC1"
+    solvent_dielectric: float = 78.5
+    solute_dielectric: float = 1
+
+
+class ForcefieldSettings(SettingsBaseModel):
+    """Settings for the forcefield.
+    """
+
+    # Metadata
+    date: Optional[date]
+    author: Optional[str]
+
+    # These should also allow None
+    vdW: VdWSettings
+    electrostatics: ElectrostaticSettings
+    gbsa: GBSASettings
+
+
+class ThermoSettings(SettingsBaseModel):
+    """Settings for thermodynamic parameters.
+
+    No checking is done to ensure a valid thermodynamic ensemble is possible.
+    """
+
+    temperature: FloatQuantity["kelvin"] = Field(None, description="Simulation temperature, default units kelvin")
+    pressure: FloatQuantity["standard_atmosphere"] = Field(None, description="Simulation pressure, default units standard atmosphere (atm)")
+    volume: FloatQuantity["nm**3"] = Field(None, description="Simulation volume, default units are cubic nanometres")
+    ph: Union[PositiveFloat, None] = Field(None, description="Simulation pH, only used if MD engine supports constant pH")
```

Comment:
I think stating things are only used if supported applies to literally every field here, just drop it.